app/src/componentLibrary/dropdown/dropdownOption/dropdownOption.jsx

```diff
@@ -43,28 +41,31 @@ export const DropdownOption = ({
       title={(disabled && title) || undefined}
       onClick={onChangeHandler}
     >
-      <div className={cx('single-option')}>{label}</div>
+      <div className={cx('single-option', { 'sub-option': !!groupRef })}>
+        {render ? render(props) : label}
```

Comment:
Please rename to `renderGroup`

Reply:
Actually it's not. Render represent single option
Visually group/sub-group manage above via groupRef property

Reply:
Ok